datalake/src/main/java/com/sequenceiq/datalake/service/sdx/flowcheck/FlowResponse.java

```diff
@@ -0,0 +1,33 @@
+package com.sequenceiq.datalake.service.sdx.flowcheck;
+
+public class FlowResponse {
+    private String currentState;
+
+    private String triggeredEvent;
+
+    private String flowType;
+
+    public String getCurrentState() {
+        return currentState;
+    }
+
+    public void setCurrentState(String currentState) {
+        this.currentState = currentState;
+    }
+
+    public String getTriggeredEvent() {
+        return triggeredEvent;
+    }
+
+    public void setTriggeredEvent(String triggeredEvent) {
+        this.triggeredEvent = triggeredEvent;
+    }
+
+    public String getFlowType() {
+        return flowType;
+    }
+
+    public void setFlowType(String flowType) {
+        this.flowType = flowType;
+    }
+}
```

Comment:
Why do we need to define a new entity? Why not use flowCheckResponse?

Reply:
@kkalvagadda1 Aww you are right! Just deleted flowResponse and use flowCheckResponse!